### PR TITLE
Fix jit-format for Windows

### PR DESCRIPTION
**Pull request by kunalspathak**
jit-format was broken on Windows after https://github.com/dotnet/runtime/pull/43651 was merged. Fixed it to use `Windows` instead of `Windows_NT`. This will break for people who still haven't merged the changes for `Windows_NT` -> `Windows`, but it is OK to break them because eventually using `Windows` is the right thing.

### CODE REVIEW

**Comment by kunalspathak** [comment]
@dotnet/jit-contrib

**Review by sandreenko — APPROVED** [review]
LGTM, do you know if windows formatting job is still broken in ci?

**Review by jashook — APPROVED** [review]

**Comment by kunalspathak** [comment]
> LGTM, do you know if windows formatting job is still broken in ci?

I am not sure. May be after I merge this PR, I can revert https://github.com/dotnet/runtime/pull/40035 and https://github.com/dotnet/runtime/pull/40517 and trigger a CI run?

**Comment by kunalspathak** [comment]
> > LGTM, do you know if windows formatting job is still broken in ci?
> 
> I am not sure. May be after I merge this PR, I can revert [dotnet/runtime#40035](https://github.com/dotnet/runtime/pull/40035) and [dotnet/runtime#40517](https://github.com/dotnet/runtime/pull/40517) and trigger a CI run?

@sandreenko - Sent out https://github.com/dotnet/runtime/pull/45015

**Pull request merged**
